pkg/api/handlers/compat/containers.go

```diff
@@ -395,6 +395,15 @@ func LibpodToContainer(l *libpod.Container, sz bool) (*handlers.Container, error
 	}, nil
 }
 
+func convertSecondaryIPPrefixLen(input *define.InspectNetworkSettings, output types.NetworkSettings) {
```

Comment:
output should be a pointer if you modify it in place:
```suggestion
func convertSecondaryIPPrefixLen(input *define.InspectNetworkSettings, output *types.NetworkSettings) {
```

Reply:
Done.